src/net_processing.cpp

```diff
@@ -1463,10 +1466,22 @@ static void ProcessPriorityRequests() {
 
             blocksToDownloadFirst.pop_front();
         }
-        else {
+        // SPV filtercp block
+        else if (r.pindex->nStatus & BLOCK_VALID_MASK)
```

Comment:
Is BLOCK_VALID_MASK the best way to control this or can it cause other issues for SPV sync?